R/create_service_costs.R

```diff
@@ -0,0 +1,71 @@
+#' Create seperate costs for each service type
+#'
+#' @param data A data frame
+#'
+#' @return A data frame with 14 additional variables, 13 costs and one flag for operations
+#' @export
```

Comment:
Remove export for non top level functions

Reply:
Done, all functions in main lookup script